packages/vite/CHANGELOG.md

```diff
@@ -52,19 +12,17 @@ See beta docs at the [Lighting CSS guide](https://main.vitejs.dev/guide/features
 
 [esbuild 0.18](https://github.com/evanw/esbuild/blob/main/CHANGELOG.md#0180) contains backwards-incompatible changes to esbuild's handling of `tsconfig.json` files. We think they shouldn't affect Vite users, you can review [#13525](https://github.com/vitejs/vite/issues/13525) for more information.
 
+### Templates for Solid and Qwik in create-vite
 
+New starter templates have been added to [create-vite](https://vitejs.dev/guide/#scaffolding-your-first-vite-project) for [Solid](https://www.solidjs.com/) and [Qwik](https://qwik.builder.io/). Try them online at [vite.new/solid-ts](https://vite.new/solid-ts) and [vite.new/qwik-ts](https://vite.new/qwik-ts).
```

Comment:
That reminds me we need to update the supported list too: https://vitejs.dev/guide/#trying-vite-online

Reply:
Same happened to me, I did a quick commit here https://github.com/vitejs/vite/commit/95f092ffb55e414ab7937d7aa1b68db91e4626fe

Reply:
Ha nice catch